src/view/tile_chunk.rs

```diff
@@ -97,7 +97,6 @@ impl Default for ChunkBundle {
     fn default() -> Self {
         Self {
             map_parent: Handle::default(),
-            chunk: TileMapChunk::default(),
```

Comment:
I think we should keep a TileMapChunk marker component at least. Otherwise it's hard to identify this entity apart the others in the world.